tests/Traits/ValidCodeAnalysisTestTrait.php

```diff
@@ -76,6 +76,20 @@ public function testValidCode(
         $codebase->enterServerMode();
         $codebase->config->visitPreloadedStubFiles($codebase);
 
+        // avoid MethodSignatureMismatch for __unserialize/() when extending DateTime
+        if (version_compare(PHP_VERSION, '8.2', '>')) {
```

Comment:
Did you mean `>= 8.2`? Or was the change triggering the issue introduces in 8.2.1?

Reply:
Was ok from the tests, but we can use PHP_VERSION_ID.
```
php -r "echo version_compare('8.2.0', '8.2', '>');"
1
php -r "echo version_compare('8.2.0alpha1', '8.2', '>');"
1
php -r "echo version_compare('8.2.0alpha1', '8.2', '>=');"
1
php -r "echo version_compare('8.2.1', '8.2', '>');"
1
```